client/src/components/Button/index.tsx

```diff
@@ -0,0 +1,121 @@
+/**  @jsxImportSource @emotion/react */
+"use client";
+
+import { css } from "@emotion/react";
+import { Button as MuiButton } from "@mui/material";
+import typography from "@/styles/typography";
+
+type ButtonProps = Parameters<typeof MuiButton>[0];
+
+function black(props: ButtonProps) {
```

Comment:
```suggestion
function Black(props: ButtonProps) {
```

`black`, `white`, `capsule` 하나 하나도 컴포넌트인 만큼 PascalCase로 이름을 쓰는 게 좋아보입니다!

Reply:
아래도 마찬가지